modules/mobile/spec/request/threads_request_spec.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../support/iam_session_helper'
+require_relative '../support/mobile_sm_client_helper'
```

Comment:
with my VCR PR just being merged, these should be pointing to the helpers folder.

Reply:
Ooo good catch, thanks. I just merged master so I'll make sure that passes